docs/src/increase-observability/metrics/_index.md

```diff
@@ -18,6 +18,10 @@ and their average for the Production environment.
 
 ![A screenshot of what the metrics dashboard displays for Dedicated environments](/images/metrics/all-dedicated.png "0.45")
 
+[Dedicated Generation 3 environments](../../dedicated-gen-3/overview.md): each of the three hosts and their average.
+
+![A screenshot of what the metrics dashboard displays for Dedicated Generation 3 environments](/images/metrics/all-dedicated-gen3.png "0.45")
```

Comment:
:lady_beetle: The image is broken and doesn't seem to be [in the repo](https://github.com/platformsh/platformsh-docs/tree/dg3-metrics/docs/static/images/metrics).

Reply:
The PR is on hold, but that issue hasn't been solved.